### PR TITLE
Adds case study:  three-js-case-study.md

### PR DESCRIPTION
**Pull request by StormSkoglund**
The following case study was a collaboration between Marius Kjesbu Bjeglerud, Weronika Karczmarczyk & Alex Storm Skoglund

Division of tasks:

Marius - Introduction & Brief History
Weronika - Main Features & Feature Table
Alex - Market Comparison & Getting started
Shared tasks - Conclusion & References

### CODE REVIEW

**Review by Roselynepj1 — COMMENTED** [review]
I am glad you managed to solve the problems yesterday. You did a great job, you case study is impressive. Keep it up,

File: src/content/docs/three-js/three-js-case-study.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+Three.js can be used to make 3D models, animations and even entire virtual worlds, the library supports a wide range of file formats such as (.glft and .obj) so its easy to import different 3D assets.
+The library has a lot of documentation and is being updated often by a active community, making Three.js an excellent tool for both beginners and experienced developers in the field of 3D development.
+The first bigger companies that started using Three.js was Google Search and apple also used Three.js to render many of their product pages [wikipedia.org].
+
+## Brief History
+
+- 2002: The idea of Three.js was started by Ricardo Cabello(Mr.doob) with his involvement with the demoscene. (The demoscene is an computer art subculture focused on producing self-contained programs that produce audiovisual presentations)
+- 2009: The project was ported to JavaScript from ActionScript. (it was called Three.as before porting over to javascript)
```
Comment (cnnrbrn) [inline src/content/docs/three-js/three-js-case-study.md:17]:
```suggestion
- 2009: The project was ported to JavaScript from ActionScript. (it was called Three.as before porting over to JavaScript)
```

File: src/content/docs/three-js/three-js-case-study.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+Some of these frameworks have smaller communities and are still in the early stages of development. For instance, Whitestorm.js is a newer framework that incorporates the latest ECMAScript features, but it hasn’t gained a huge following yet [slant.co].
+
+While only having a small community, Babylon.js, is one of the main competitors to Three.js and offers support for Apple's AR goggles. Babylon.js presents a more professional appeal and delivers visual fidelity that surpasses both Three.js and Whitestorm [slant.co] [babylonjs.com].
+
+PlayCanvas, a third competitor, claims to have excellent cross-platform compatibility. It boasts similar features as both Three.js and Babylon.js. It provides users with many tools geared towards game development, although some of its features are hidden behind a paid subscription [slant.co] [playcanvas.com].
+
+One of Three.js’ main strengths appears to be the great documentation provided by the developers and also its large number of official plugins and tutorials [slant.co].
+
+###The following table compares Three.js with its main competitors across various key points:
```
Comment (cnnrbrn) [inline src/content/docs/three-js/three-js-case-study.md:79]: Should this be a heading?

File: src/content/docs/three-js/three-js-case-study.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+| Rich         | Rich       | Medium     | Rich\*\*\* |
+|              |            |            |            |
+|              |
+| Fidelity:    | Fidelity:  | Fidelity   | Fidelity   |
+| Mid/high\*\* | High       | low        | High       |
+|              |            |            |            |
+| License: N   | License: N | License: N | License: Y |
+
+Y = Yes, N = No
+\*Three.js does not include its own physics engine, but is fully compatible with external physics frameworks for JavaScript, as explained in this article: "https://en.threejs-university.com/2021/08/24/using-physics-in-three-js-with-ammo-jsintroduction/" [threejs-university.com].
```
Comment (cnnrbrn) [inline src/content/docs/three-js/three-js-case-study.md:100]: Convert the link here (and the others) to a proper markdown link, [see](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links)

File: src/content/docs/three-js/three-js-case-study.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+NB: PlayCanvas Offers a free license, as long as the project-size is kept below a certain size [playcanvas.com].
+
+### Market Comparison Conclusion
+
+Each of the aforementioned frameworks has its own strengths and weaknesses when compared to each other. The choice of which framework to use for displaying three-dimensional content to an audience is ultimately up to you. However, Three.js is a strong contender due to its large community, free-to-use features, and official plugins.
+
+## Getting Started
+
+## Getting started and setting up the project.
```
Comment (cnnrbrn) [inline src/content/docs/three-js/three-js-case-study.md:114]: Seems like a double heading?

File: src/content/docs/three-js/three-js-case-study.md (hunk 2 above)
Comment (StormSkoglund) [reply src/content/docs/three-js/three-js-case-study.md:79]: I have now deleted the subheading , making it a part of the text instead.

File: src/content/docs/three-js/three-js-case-study.md (hunk 4 above)
Comment (StormSkoglund) [reply src/content/docs/three-js/three-js-case-study.md:114]: I have now removed the first heading. I have also rewritten the links with proper Markdown syntax. I will will push my changes soon!

**Comment by cnnrbrn** [comment]
The links are still not formatted correctly.

**Comment by StormSkoglund** [comment]
Hi

> The links are still not formatted correctly.

Hi Connor. I have made a push, this time he links should be correctly formatted. Please let me know if this is not the case. Kind regards

**Review by cnnrbrn — APPROVED** [review]

**Pull request merged**